lib/media/mp4_ttml_parser.js

```diff
@@ -45,7 +47,7 @@ shaka.media.Mp4TtmlParser =
     // mdat box found, use TtmlTextParser to parse the content
     return shaka.media.TtmlTextParser(
         reader.readBytes(boxSize - 8).buffer, offset,
-        segmentStartTime, segmentEndTime);
+        segmentStartTime, segmentEndTime, null);
```

Comment:
It's less confusing if you pass the value of useRelativeCueTimestamps, the other parsers can ignore it.